lib/uk/ac/ebi/interpro/InterProScan.groovy

```diff
@@ -474,35 +475,88 @@ class InterProScan {
     }
 
     static void printHelp(appsConfig) {
+        def optionsMap = this.PARAMS
+            .findAll { it.description }
+            .collectEntries { opt ->
+                [
+                    (opt.name): [
+                        description: opt.description,
+                        metavar    : opt.metavar
+                    ]
+                ]
+            }
+
         def result = new StringBuilder()
-        result << "Usage: nextflow run ebi-pf-team/interproscan6 -profile <PROFILE> --input <FASTA> --datadir <DATADIR> \n\n"
+        result << "Usage: nextflow run ebi-pf-team/interproscan6 -profile <PROFILE> --input <FASTA> [--datadir <DATADIR>] [options]\n\n"
```

Comment:
Now that we support bare metal execution isn't profiles also technically optional and the only argument that is always mandatory is `--input`? So it could be something like:
```suggestion
        result << "Usage: nextflow run ebi-pf-team/interproscan6 --input <FASTA> [--datadir <DATADIR> -profile <PROFILE>] [options]\n\n"
```
Or are you aiming to push for users to use containers?